### PR TITLE
BigTable: add 'client_options' support to Client

### PR DESCRIPTION
**Pull request by Emar-Kar**
IPR: 8475

### CODE REVIEW

File: bigtable/setup.py [hunk 1]
```diff
@@ -20,72 +20,71 @@
 
 # Package metadata.
 
-name = 'google-cloud-bigtable'
-description = 'Google Cloud Bigtable API client library'
-version = '1.0.0'
+name = "google-cloud-bigtable"
```
Comment (IlyaFaer) [inline bigtable/setup.py]: Better revert this changes. It's a stray commit, and, as Chris Wilcox said, they are using robots to chang `setup.py` files (at least, some parts of it)

File: bigtable/google/cloud/bigtable/client.py [hunk 2]
```diff
@@ -109,6 +113,11 @@ class Client(ClientWithProject):
         you only need to set this if you're developing your own library
         or partner tool.
 
+    :type client_options: :class:`~google.api_core.client_options.ClientOptions`
+        or :class:`dict`
+    :param client_options: (Optional) Client options used to set user options
+        on the client. API Endpoint should be set through client_options.
```
Comment (IlyaFaer) [inline bigtable/google/cloud/bigtable/client.py]: client_options mast be marked with ``

File: bigtable/setup.py (hunk 1 above)
Comment (Emar-Kar) [reply bigtable/setup.py]: Each time I modified `client_options` in over [libraries](https://github.com/googleapis/google-cloud-python/pull/8999#discussion_r315803265) it was requested to update the `core` version. The `setup.py` file was updated due to the `black`.

File: bigtable/google/cloud/bigtable/client.py (hunk 2 above)
Comment (Emar-Kar) [reply bigtable/google/cloud/bigtable/client.py]: I checked all over libraries, they are not marked there.

Comment (IlyaFaer) [reply bigtable/google/cloud/bigtable/client.py]: Looks like Bu Sun Kim didn't wrote such marking, but usually source code is marked like this
image
Well, on your will
